test/server/cards/02_SHD/units/HunterOfTheHaxionBrood.spec.ts

```diff
@@ -0,0 +1,44 @@
+describe('Hunter of the Haxion Brood', function () {
+    integration(function (contextRef) {
+        describe('Hunter of the Haxion Brood\'s ability', function () {
+            beforeEach(function () {
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
+                        hand: ['hunter-of-the-haxion-brood'],
+                    },
+                    player2: {
+                        groundArena: ['hylobon-enforcer']
+                    }
+                });
+            });
+
+            it('should have Shield because opponent has a unit with bounty.', function () {
+                const { context } = contextRef;
+                context.player1.clickCard(context.hunterOfTheHaxionBrood);
+                expect(context.hunterOfTheHaxionBrood).toHaveExactUpgradeNames(['shield']);
+            });
+        });
+
+        // TODO ADD TEST WITH BOUNTY UPGRADES
```

Comment:
I think we can remove this, I will add some tests of "gain Bounty" to a generic Bounty.spec.ts in my PR which will cover it.

Same for Trandoshan Hunter